src/main/java/seedu/guilttrip/logic/commands/remindercommands/DisplayPopupCommand.java

```diff
@@ -0,0 +1,32 @@
+package seedu.guilttrip.logic.commands.remindercommands;
+
+import seedu.guilttrip.logic.CommandHistory;
+import seedu.guilttrip.logic.commands.Command;
+import seedu.guilttrip.logic.commands.CommandResult;
+import seedu.guilttrip.logic.commands.exceptions.CommandException;
+import seedu.guilttrip.model.Model;
+
+/**
+ * Displays popup for GeneralReminder.
+ */
+public class DisplayPopupCommand extends Command {
+    public static final String COMMAND_WORD = "popup";
```

Comment:
Same here. You already have a great description at the javadoc for this class, can just copy.